extension/textext/requirements_check.py

```diff
@@ -519,34 +590,45 @@ def find_pstoedit(self, version=None):
                     "pstoedit=%s is not found (but pstoedit=%s is found)" % (version, found_version)])
         return RequirementCheckResult(None, ["Can't determinate pstoedit version"])
 
-    def find_executable(self, executable_name):
```

Comment:
revert changes to this function, all system-specific changes would be in `_find_executable_in_path`